.circleci/config.yml

```diff
@@ -426,9 +426,10 @@ jobs:
       - run:
           name: get wasmtime
           command: |
-            wget https://github.com/CraneStation/wasmtime/releases/download/dev/wasmtime-dev-x86_64-linux.tar.xz
-            tar -xf wasmtime-dev-x86_64-linux.tar.xz
-            cp wasmtime-dev-x86_64-linux/wasmtime ~/vms
+            # use a pinned version due to https://github.com/bytecodealliance/wasmtime/issues/714
+            wget https://github.com/bytecodealliance/wasmtime/releases/download/v0.8.0/wasmtime-v0.8.0-x86_64-linux.tar.xz
+            tar -xf wasmtime-*-x86_64-linux.tar.xz
+            cp wasmtime-*-x86_64-linux/wasmtime ~/vms
```

Comment:
I wouldn't use wild card here, just duplicate the version, or use an environment variable.  e.g:

```
VERSION=0.8.0
wget https://github.com/bytecodealliance/wasmtime/releases/download/v0.8.0/wasmtime-v$VERSION-x86_64-linux.tar.xz
cp wasmtime-dev-x86_64-linux/wasmtime ~/vms
tar -xf wasmtime-$VERSION-x86_64-linux.tar.xz
cp wasmtime-$VERSION-x86_64-linux/wasmtime ~/vms
```

Reply:
Changed to an env var, thanks.